src/movies.js

```diff
@@ -1,16 +1,89 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies) {
+    const allDirectors = movies.map(item => item.director)
+    return allDirectors
+}
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+const howManyMovies = (movies) => {
+    let spielbergFilms = movies.filter(
+      films => films.director.includes("Steven Spielberg"))
+
+    let dramaFilms = spielbergFilms.filter(
+      films => films.genre.includes("Drama"))
+    return dramaFilms.length
+  }
+  
```

Comment:
Another way to write this 

`function howManyMovies(someMovies) {
 
 return someMovies.filter(
    eachMovie =>
      eachMovie.director === 'Steven Spielberg' &&
      eachMovie.genre.includes('Drama')
  ).length;
}`